scripts/spack/configs/toss_4_x86_64_ib/packages.yaml

```diff
@@ -0,0 +1,280 @@
+    externals:
+    - spec: diffutils@3.3
+      prefix: /usr
+  elfutils:
+    buildable: false
+    externals:
+    - spec: elfutils@0.176
+      prefix: /usr
+  epoxy:
+    buildable: false
+    externals:
+    - spec: epoxy@1.5.2
+      prefix: /usr
+  findutils:
+    buildable: false
+    externals:
+    - spec: findutils@4.5.11
+      prefix: /usr
+  gettext:
+    buildable: false
+    externals:
+    - spec: gettext@0.19.8.1
+      prefix: /usr
+  ghostscript:
+    buildable: false
+    externals:
+    - spec: ghostscript@9.25
+      prefix: /usr
+  groff:
+    buildable: false
+    externals:
+    - spec: groff@1.22.2
+      prefix: /usr
+  graphviz:
+    buildable: false
+    externals:
+    - spec: graphviz@2.30.1
+      prefix: /usr
+  libtool:
+    buildable: false
+    externals:
+    - spec: libtool@2.4.2
+      prefix: /usr
+  libunwind:
+    buildable: false
+    externals:
+    - spec: libunwind@8.0.1
+      prefix: /usr
+  libx11:
+    buildable: false
+    externals:
+    - spec: libx11@1.20.4
+      prefix: /usr
+  m4:
+    buildable: false
+    externals:
+    - spec: m4@1.4.16
+      prefix: /usr
+  perl:
+    buildable: false
+    externals:
+    - spec: perl@5.16.3
+      prefix: /usr
+  pkg-config:
+    buildable: false
+    externals:
+    - spec: pkg-config@0.27.1
+      prefix: /usr
+  readline:
+    buildable: false
+    externals:
+    - spec: readline@6.2
+      prefix: /usr
+  tar:
+    buildable: false
+    externals:
+    - spec: tar@1.26
+      prefix: /usr
+  unzip:
+    buildable: false
+    externals:
+    - spec: unzip@6.0
+      prefix: /usr
+  zlib:
+    buildable: false
+    externals:
+    - spec: zlib@1.2.7
+      prefix: /usr
+  curl:
+    buildable: false
+    externals:
+    - spec: curl@7.61.1
+      prefix: /usr
+
+# External dependencies for SCR
+  lsf:
+    buildable: False
+    externals:
+    - spec: lsf@10.1
+      prefix: /opt/ibm/spectrumcomputing/lsf/10.1
+  slurm:
+    buildable: false
+    externals:
+    - spec: slurm@20
+      prefix: /usr
+  libyogrt:
+    externals:
+    - spec: libyogrt@1.24 scheduler=lsf
+      prefix: /usr
+    - spec: libyogrt@1.24 scheduler=slurm
+      prefix: /usr
+  pdsh:
+    buildable: false
+    externals:
+    - spec: pdsh@2.33
+      prefix: /usr
+
+# Globally lock version of third party libraries
+  conduit:
+    version: [0.7.2axom]
+    variants: ~shared~test
+  hdf5:
+    variants: ~shared~mpi
+  hypre:
+    version: [2.20.0]
+  mfem:
+    version: [4.2.0]
+  raja:
+    variants: ~shared~examples~exercises
+  umpire:
+    version: [6.0.0axom]
+    variants: ~shared~examples
+
+# Globally lock in versions of Devtools
+  cmake:
+    version: [3.21.1]
+    buildable: false
+    externals:
+    - spec: cmake@3.21.1
+      prefix: /usr/tce/packages/cmake/cmake-3.21.1
+  cppcheck:
+    version: [1.87]
+    buildable: false
+    externals:
+    - spec: cppcheck@1.87
+      prefix: /collab/usr/gapps/axom/devtools/toss_3_x86_64_ib/latest/cppcheck-1.87
```

Comment:
All of these paths are incorrect and point to the non-cray devtools.  Just remove them unless you are planning on building them for this platform.